watertap/property_models/unit_specific/activated_sludge/asm1_properties.py

```diff
@@ -513,35 +542,3 @@ def define_display_vars(self):
 
     def get_material_flow_basis(self):
         return MaterialFlowBasis.mass
-
-    def calculate_scaling_factors(self):
-        # Get default scale factors and do calculations from base classes
-        super().calculate_scaling_factors()
-
-        # No constraints in this model as yet, just need to set scaling factors
-        # for expressions
-        sf_F = iscale.get_scaling_factor(self.flow_vol, default=1e2, warning=True)
-        sf_T = iscale.get_scaling_factor(self.temperature, default=1e-2, warning=True)
-
-        # Mass flow and density terms
-        for j in self.component_list:
-            if j == "H2O":
-                sf_C = pyo.value(1 / self.params.dens_mass)
-            elif j == "S_ALK":
-                sf_C = 1e-1 * iscale.get_scaling_factor(
-                    self.alkalinity, default=1, warning=True
-                )
-            else:
-                sf_C = iscale.get_scaling_factor(
-                    self.conc_mass_comp[j], default=1e2, warning=True
-                )
-
-            iscale.set_scaling_factor(self.material_flow_expression[j], sf_F * sf_C)
-            iscale.set_scaling_factor(self.material_density_expression[j], sf_C)
-
-        # Enthalpy and energy terms
-        sf_rho_cp = pyo.value(1 / (self.params.dens_mass * self.params.cp_mass))
-        iscale.set_scaling_factor(
-            self.enthalpy_flow_expression, sf_F * sf_rho_cp * sf_T
-        )
-        iscale.set_scaling_factor(self.energy_density_expression, sf_rho_cp * sf_T)
```

Comment:
Do we need to remove this at this point? We can just leave it in until we completely do away with use of `calculate_scaling_factors`. For our purposes, I would think that we just wouldn't call calculate_scaling_factors on the model.

I guess the only reason we might consider temporarily deleting this is so calculate_scaling_factors isn't called here when we call it at the flowsheet level (for the time being).

Reply:
I think we can keep these for now. I had deleted them before we discussed on Monday and just haven't added them back in yet.

Reply:
I've added these back, and I don't think there will be any problems with keeping the old `calculate_scaling_factors` around